packages/components/src/Actions/ActionDropdown/ActionDropdown.component.js

```diff
@@ -81,6 +83,26 @@ function getMenuItem(item, index, getComponent) {
 	return renderMutableMenuItem(item, index, getComponent);
 }
 
+function getDropdownToggleFromInner(innerElement) {
+	let dropdownTrigger = innerElement;
+	while (!dropdownTrigger.classList.contains('dropdown-toggle')) {
+		dropdownTrigger = dropdownTrigger.parentElement;
+	}
+	return dropdownTrigger;
+}
+
+function getDropdownContainer(dropdownElement) {
+	let dropdownContainer = dropdownElement;
+	do {
+		dropdownContainer = dropdownContainer.parentElement;
+	} while (
+		dropdownContainer &&
+		dropdownContainer.tagName !== 'BODY' &&
```

Comment:
I have a doubt if we should check with dropdownContainer.tagName.toUpperCase().

Reply:
No need to. From MDN : 

<img width="675" alt="capture d ecran 2018-06-07 a 10 49 55" src="https://user-images.githubusercontent.com/10761073/41088743-93f9ace0-6a40-11e8-8497-a5bfa9dc9736.png">